src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -459,6 +464,32 @@ export default class ExportImportService
             : [];
     }
 
+    private async getUnsupportedDependencies(
```

Comment:
maybe getMissingDependencies to be consistent with the names you use inside this method?